src/test/java/org/mskcc/cmo/metadb/PortedLimsRestCmoLabelGenerationTest.java

```diff
@@ -56,7 +56,7 @@ public void testPatientNoExistingSamples() throws Exception {
                 SpecimenType.XENOGRAFT, NucleicAcid.DNA);
 
         String cmoId = cmoLabelGeneratorService.generateCmoSampleLabel(requestId, sample, new ArrayList<>());
-        Assert.assertEquals("C-1235-X001-d", cmoId);
+        Assert.assertEquals("C-1235-X001-d01", cmoId);
```

Comment:
Can we have a d01?  Maybe I'm reading the spec wrong, but don't we go from -d to -d02?